configure.in

```diff
@@ -1,6 +1,5 @@
 dnl Process this file with autoconf to produce a configure script.
-
-AC_PREREQ(2.58)
+AC_PREREQ([2.71])
```

Comment:
As mentioned, I think we can use 2.69 here without any other changes to this file. Could you please check that it indeed works and make this change? 2.69 is the version in Debian 11 (bullseye) and I don't see any reason to drop support for it if we don't use anything that it doesn't support.

Reply:
I'll put this idea to the test, but I suspect some things *did* change, such as the AC_PROC_CC_99 macro having been made obsolete by AC_PROG_CC since Autoconf 2.70.

Reply:
There was no problem, done!